docs/user/production-considerations/production.asciidoc

```diff
@@ -68,13 +73,23 @@ bin/kibana -c config/instance2.yml
 === Accessing multiple load-balanced {kib} clusters
 
 To access multiple load-balanced {kib} clusters from the same browser,
-set `xpack.security.cookieName` in the configuration.
+explicitly set `xpack.security.cookieName` to the same value in the {kib} configuration
+of each {kib} instance.
+
+Each {kib} cluster must have a different value of `xpack.security.cookieName`.
+
 This avoids conflicts between cookies from the different {kib} instances.
 
-In each cluster, {kib} instances should have the same `cookieName`
-value. This will achieve seamless high availability and keep the session
+This will achieve seamless high availability and keep the session
 active in case of failure from the currently used instance.
 
+[float]
+[[reporting-load-balanced-kibana]]
+=== Reporting and multiple load-balanced {kib} clusters
+
+Depending on the load balancing configuration, you might need to specify 
+`xpack.reporting.kibanaServer.*` settings to use *Reporting*.
```

Comment:
I'm not familiar with this setting, but maybe @tsullivan can confirm that this makes sense

Reply:
We should not promote using the `xpack.reporting.kibanaServer`, unless there is no way to open a local connection to Kibana for capturing the screenshots. I think it's best to leave this out.

Related issue: https://github.com/elastic/kibana/issues/86425